4-js/index.js

```diff
@@ -0,0 +1,330 @@
+
+addUserBtn.addEventListener('click', () => {
+  const arrFormUserInput = document.getElementsByClassName('form-user-input');
+  formUser.classList.remove('is-hide');
+  createUserBtn.classList.remove('is-hide');
+  titleNewUser.classList.remove('is-hide');
+  createUserBtn.setAttribute('disabled', 'disabled');
+  createUserBtn.classList.add('is-translucent');
+  for (let i = 0; i < arrFormUserInput.length; i++) {
+    arrFormUserInput[i].addEventListener('keyup', formCheck);
+  }
+});
+
+createUserBtn.addEventListener('click', () => {
+  createUser();
+  formUser.classList.add('is-hide');
+  createUserBtn.classList.add('is-hide');
+  titleNewUser.classList.add('is-hide');
+  userCompany.value = '';
+  userName.value = '';
+  userAddress.value = '';
+  userCity.value = '';
+  userCountry.value = '';
+});
+
+allDelete.addEventListener('click', () => {
+  deleteSelectedUsers();
+});
+
+mainCheckbox.addEventListener('click', () => {
+  for (let i = 0; i < arrUserCheckbox.length; i++) {
+    if (mainCheckbox.checked !== arrUserCheckbox[i].checked) {
+      arrUserCheckbox[i].checked = !arrUserCheckbox[i].checked;
+    }
+  }
+  allDelete.classList.toggle('is-hide');
+});
+
+editUserBtn.addEventListener('click', () => {
+  editUser();
+  userCompany.value = '';
+  userName.value = '';
+  userAddress.value = '';
+  userCity.value = '';
+  userCountry.value = '';
+  formUser.classList.add('is-hide');
+  editUserBtn.classList.add('is-hide');
+  titleEditUser.classList.add('is-hide');
+});
+
+function findArrLenghtUsers() {
+  sendRequsest('GET',requestUrl).then((data) => {
+    createPaginator(data.length)
+  })
+  .catch((err) => console.log(err));
+}
+
+function createPaginator(length) {
+  const quantityPagination = Math.ceil(length / 10);
+  for (let i = 1; i <= quantityPagination; i++) {
+    let button = document.createElement('button');
+    button.innerText = i;
+    button.name = i * 10;
+    button.classList.add('paginator-number');
+    paginator.appendChild(button);
+    if (i === 1) {
+      button.classList.add('active');
+    }
+    button.addEventListener('click',showPagination);
+  }
+  getUsersInfo('_start=' + (quantityPagination * 10 - 10) + '&_limit=50');
+}
+
+function showPagination() {
+  getUsersInfo('_start=' + (paginations.length * 10 - this.name) + '&_limit=50');
+  changeStylePagination(this);
+}
+
+function changeStylePagination(el) {
+  const arrPaginations = document.getElementsByClassName('paginator-number');
+  for (let i = 0; i < arrPaginations.length; i++) {
+    arrPaginations[i].classList.remove('active');
+  }
+  el.classList.add('active');
+}
+
+function addInputListener() {
+    for (let i = 0; i < inputFilter.length; i++) {
+        inputFilter[i].addEventListener('keyup',e => {
+            if (e.keyCode >= 48&&e.keyCode <= 90||e.keyCode === 8) {
+                let resultSort = '';
+                if (companyInput.value !== '') {
+                    resultSort += 'company_like=' + companyInput.value + '&&';
```

Comment:
Instead of concatenation you can use a template string. And why do you need add `&&`?

Reply:
I'm adding &&, to use several sorting fields at once.